android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Wed Mar 18 16:06:58 IST 2020
-user=
+#Mon May 14 23:39:14 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```

Comment:
Why?

Reply:
I have done changes as per updated SDK which contains this distribution url only.